packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,58 @@
+  "nutanix_redeploy_custom_configuration_prism_central_type_description": "Le champ « Type » détermine le niveau de disponibilité de la machine virtuelle (VM) Prism Central que vous souhaitez appliquer. Chaque VM consomme une partie des ressources de votre cluster. Le type « Alone » signifie qu'une seule VM est déployée. Le type « Scale-out » signifie que trois machines sont déployées sur votre cluster pour gérer Prism Central. Cela affecte la capacité d'évolution des VM ainsi que la résilience des systèmes.",
+  "nutanix_redeploy_custom_configuration_prism_central_virtual_ip": "IP virtuelle (VIP)",
+  "nutanix_redeploy_custom_configuration_prism_central_virtual_ip_description": "VIP si l'IP flottante est utilisée pour exposer Prism Central. Cette IP sera configurée dans le back-end des IP publiques Load Balancer de votre cluster.",
+  "nutanix_redeploy_custom_configuration_prism_central_ip": "IP",
+  "nutanix_redeploy_custom_configuration_prism_central_ip_description": "IP privées des VM Prism Central.",
+  "nutanix_redeploy_custom_configuration_replication": "Replication",
+  "nutanix_redeploy_custom_configuration_replication_description": "Vous pouvez opter pour des données (VM et Oplog) répliquées deux ou trois fois au sein de votre cluster. Une plus grande réplication permet une meilleure tolérance aux incidents, mais consommera davantage d’espace de stockage.",
+  "nutanix_redeploy_custom_configuration_erasure_coding": "Erasure Coding",
+  "nutanix_redeploy_custom_configuration_erasure_coding_description": "Vous pouvez activer la fonction « Erasure Coding ». Celle-ci augmentera la capacité de stockage effective, mais n’est pas recommandée en dessous de 4 nœuds.",
+  "nutanix_redeploy_custom_configuration_prism_element_vip": "Prism element Virtual IP (VIP):",
+  "nutanix_redeploy_custom_configuration_prism_element_vip_description": "VIP si l'IP flottante est utilisée pour exposer Prism Central. Cette IP sera configurée dans le back-end de l'IP publique Load Balancer de votre cluster.",
+  "nutanix_redeploy_custom_configuration_prism_element_gateway_cidr": "Gateway CIDR",
+  "nutanix_redeploy_custom_configuration_prism_element_gateway_cidr_description": "Désignez le réseau privé qui sera utilisé pour le déploiement du cluster. La passerelle de ce réseau sera utilisée comme OVH Gateway VM IP.",
+  "nutanix_redeploy_custom_configuration_prism_element_aos": "AOS",
+  "nutanix_redeploy_custom_configuration_prism_element_aos_description": "AOS est le système d'exploitation Nutanix. Choisissez la version d'AOS que vous souhaitez installer sur l'ensemble des noeuds composant votre cluster.",
+  "nutanix_redeploy_custom_configuration_replication_rf2": "Facteur de réplication = 2 (RF2)",
+  "nutanix_redeploy_custom_configuration_replication_rf3": "Facteur de réplication = 3 (RF3)",
+  "nutanix_redeploy_custom_configuration_prism_central_type_alone": "Alone",
+  "nutanix_redeploy_custom_configuration_prism_central_type_scale": "Scale",
```

Comment:
**question:** Should it be translated? If it's not supposed to be translated, then it's a constant.